2022-05-01 02:44:05,518 - ERROR[etl_loader.RawDataExtract][extract.get_data] - Unable to read file raw_zone\trips_2018-05.csv.CSV
2022-05-01 02:44:05,518 - ERROR[etl_loader.RawDataExtract][extract.get_data] - Unable to read file raw_zone\trips_2018-06.csv.CSV
2022-05-01 02:44:29,877 - ERROR[etl_loader][elt_loader.get_all] - Export trips with errorDuplicate names are not allowed.
2022-05-01 02:46:15,119 - ERROR[etl_loader][elt_loader.get_all] - Export trips with errorDuplicate names are not allowed.
2022-05-01 02:54:16,730 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error Duplicate names are not allowed.
2022-05-01 02:54:33,885 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error Duplicate names are not allowed.
2022-05-01 03:01:53,540 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error Duplicate names are not allowed.
2022-05-01 03:04:23,361 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error Duplicate names are not allowed.
2022-05-01 03:04:44,471 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error Duplicate names are not allowed.
2022-05-01 03:27:50,943 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error Duplicate names are not allowed.
2022-05-01 03:32:20,665 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error first argument must be an iterable of pandas objects, you passed an object of type "DataFrame"
2022-05-01 03:33:03,618 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error 'origin_coord'
2022-05-01 03:34:43,950 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error 'origin_coord'
2022-05-01 03:36:24,888 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error 'origin_coord'
2022-05-01 03:37:11,168 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error Duplicate names are not allowed.
2022-05-01 03:42:17,896 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error Must pass 2-d input. shape=(2, 100, 5)
2022-05-01 03:52:37,917 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error Duplicate names are not allowed.
2022-05-01 03:59:10,951 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error 'DataFrame' object has no attribute 'groupBy'
2022-05-01 04:01:02,109 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error No axis named origin_coord2 for object type DataFrame
2022-05-01 04:04:02,562 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error replace() got an unexpected keyword argument 'inplace'
2022-05-01 04:04:48,173 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error missing ), unterminated subpattern at position 6
2022-05-01 04:08:22,585 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error No axis named origin_coord2 for object type DataFrame
2022-05-01 04:11:48,316 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error No axis named origin_coord2 for object type DataFrame
2022-05-01 04:12:22,831 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error No axis named origin_coord2 for object type DataFrame
2022-05-01 04:13:01,081 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error No axis named origin_coord2 for object type DataFrame
2022-05-01 04:15:09,614 - ERROR[etl_loader.SQLServerAccess][sql_connection.__connect_direct__] - Unable to create connection to MSSQL (SQL Authentication)('HY000', "[HY000] [MySQL][ODBC 8.0(a) Driver]Access denied for user 'jobsity'@'%' to database 'jobsity' (1044) (SQLDriverConnect); [HY000] [MySQL][ODBC 8.0(a) Driver]Access denied for user 'jobsity'@'%' to database 'jobsity' (1044)")
2022-05-01 04:15:09,646 - ERROR[etl_loader.SQLServerAccess][sql_connection.__connect_direct__] - Unable to create connection to MSSQL (SQL Authentication)('HY000', "[HY000] [MySQL][ODBC 8.0(a) Driver]Access denied for user 'jobsity'@'%' to database 'jobsity' (1044) (SQLDriverConnect); [HY000] [MySQL][ODBC 8.0(a) Driver]Access denied for user 'jobsity'@'%' to database 'jobsity' (1044)")
2022-05-01 04:15:09,646 - ERROR[etl_loader][load.load_data] - 'NoneType' object has no attribute 'cursor'
2022-05-01 04:15:09,646 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error local variable 'e' referenced before assignment
2022-05-01 04:16:15,754 - ERROR[etl_loader.SQLServerAccess][sql_connection.insert_staging_with_df] - Unable to execute INSERT in Staging  ('HY000', "[HY000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]Column count doesn't match value count at row 1 (1136) (SQLPrepare)")  target_table table_staging
2022-05-01 04:16:15,754 - ERROR[etl_loader][load.load_data] - [Errno 2] No such file or directory: 'client\\querys\\delta\\merge_query.sql'
2022-05-01 04:16:15,754 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error local variable 'e' referenced before assignment
2022-05-01 04:17:01,070 - ERROR[etl_loader.SQLServerAccess][sql_connection.insert_staging_with_df] - Unable to execute INSERT in Staging  ('HY000', "[HY000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]Column count doesn't match value count at row 1 (1136) (SQLPrepare)")  target_table table_staging
2022-05-01 04:17:01,087 - ERROR[etl_loader][load.load_data] - [Errno 2] No such file or directory: 'client\\querys\\delta\\merge_query.sql'
2022-05-01 04:17:01,087 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error local variable 'e' referenced before assignment
2022-05-01 04:17:17,305 - ERROR[etl_loader.SQLServerAccess][sql_connection.insert_staging_with_df] - Unable to execute INSERT in Staging  ('HY000', "[HY000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]Column count doesn't match value count at row 1 (1136) (SQLPrepare)")  target_table table_staging
2022-05-01 04:17:17,305 - ERROR[etl_loader][load.load_data] - [Errno 2] No such file or directory: 'querys\\merge_query.sql'
2022-05-01 04:17:17,305 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error local variable 'e' referenced before assignment
2022-05-01 04:17:37,462 - ERROR[etl_loader.SQLServerAccess][sql_connection.insert_staging_with_df] - Unable to execute INSERT in Staging  ('HY000', "[HY000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]Column count doesn't match value count at row 1 (1136) (SQLPrepare)")  target_table table_staging
2022-05-01 04:17:37,493 - ERROR[etl_loader.SQLServerAccess][sql_connection.merge_execute] - Unable to execute Merge in Staging('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'MERGE table_trips AS target\n            USING (SELECT region, origin_coord, dest' at line 1 (1064) (SQLExecDirectW)")
2022-05-01 04:17:37,493 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error local variable 'e' referenced before assignment
2022-05-01 04:18:22,383 - ERROR[etl_loader.SQLServerAccess][sql_connection.insert_staging_with_df] - Unable to execute INSERT in Staging  ('HY000', "[HY000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]Column count doesn't match value count at row 1 (1136) (SQLPrepare)")  target_table table_staging
2022-05-01 04:20:37,646 - ERROR[etl_loader.SQLServerAccess][sql_connection.insert_staging_with_df] - Unable to execute INSERT in Staging  ('HY000', "[HY000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]Column count doesn't match value count at row 1 (1136) (SQLPrepare)")  target_table table_staging
2022-05-01 04:20:56,459 - ERROR[etl_loader.SQLServerAccess][sql_connection.insert_staging_with_df] - Unable to execute INSERT in Staging  ('HY000', "[HY000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]Column count doesn't match value count at row 1 (1136) (SQLPrepare)")  target_table table_staging
2022-05-01 04:23:09,492 - ERROR[etl_loader.SQLServerAccess][sql_connection.insert_staging_with_df] - Unable to execute INSERT in Staging  ('HY000', "[HY000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]Column count doesn't match value count at row 1 (1136) (SQLPrepare)")  target_table table_staging
2022-05-01 04:30:04,074 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error "['origin_coord', 'destination_coord', 'datasource'] not found in axis"
2022-05-01 04:31:14,371 - ERROR[etl_loader.SQLServerAccess][sql_connection.insert_staging_with_df] - Unable to execute INSERT in Staging  ('HY000', "[HY000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]Column count doesn't match value count at row 1 (1136) (SQLPrepare)")  target_table table_staging
2022-05-01 04:31:14,407 - ERROR[etl_loader.SQLServerAccess][sql_connection.merge_execute] - Unable to execute Merge in Staging('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'MERGE table_trips AS target\n            USING (SELECT datetime, origin_coord1, o' at line 1 (1064) (SQLExecDirectW)")
2022-05-01 04:31:14,407 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error local variable 'e' referenced before assignment
2022-05-01 04:33:55,694 - ERROR[etl_loader.SQLServerAccess][sql_connection.insert_staging_with_df] - Unable to execute INSERT in Staging  ('HY000', "[HY000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]Column count doesn't match value count at row 1 (1136) (SQLPrepare)")  target_table table_staging
2022-05-01 04:33:55,724 - ERROR[etl_loader.SQLServerAccess][sql_connection.merge_execute] - Unable to execute Merge in Staging('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'MERGE table_trips AS target\n            USING table_staging AS source\n          ' at line 1 (1064) (SQLExecDirectW)")
2022-05-01 04:33:55,724 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error local variable 'e' referenced before assignment
2022-05-01 04:38:19,325 - ERROR[etl_loader.SQLServerAccess][sql_connection.insert_staging_with_df] - Unable to execute INSERT in Staging  ('HY000', "[HY000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]Column count doesn't match value count at row 1 (1136) (SQLPrepare)")  target_table table_staging
2022-05-01 04:38:19,371 - ERROR[etl_loader.SQLServerAccess][sql_connection.merge_execute] - Unable to execute Merge in Staging('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'MERGE table_trips AS target\n            USING table_staging AS source\n          ' at line 1 (1064) (SQLExecDirectW)")
2022-05-01 04:40:04,293 - ERROR[etl_loader.SQLServerAccess][sql_connection.insert_staging_with_df] - Unable to execute INSERT in Staging  ('HY000', "[HY000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]Column count doesn't match value count at row 1 (1136) (SQLPrepare)")  target_table table_staging
2022-05-01 04:40:04,340 - ERROR[etl_loader.SQLServerAccess][sql_connection.merge_execute] - Unable to execute Merge in Staging('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'MERGE table_trips AS target\n            USING table_staging AS source\n          ' at line 1 (1064) (SQLExecDirectW)")
2022-05-01 05:26:04,132 - ERROR[etl_loader][load.load_data] - function takes exactly 2 arguments (1 given)
2022-05-01 05:26:04,132 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error function takes exactly 2 arguments (1 given)
2022-05-01 05:26:57,650 - ERROR[etl_loader.SQLServerAccess][sql_connection.insert_staging_with_df] - Unable to execute INSERT in Staging  ('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''INSERT INTO table_staging (origin_coord1, origin_coord2, destination_coord1, de' at line 1 (1064) (SQLExecDirectW)")  target_table table_staging
2022-05-01 05:26:57,681 - ERROR[etl_loader.SQLServerAccess][sql_connection.merge_execute] - Unable to execute Merge in Staging('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'MERGE table_trips AS target\n            USING table_staging AS source\n          ' at line 1 (1064) (SQLExecDirectW)")
2022-05-01 05:27:35,087 - ERROR[etl_loader.SQLServerAccess][sql_connection.insert_staging_with_df] - Unable to execute INSERT in Staging  ('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''INSERT INTO jobsity.table_staging (origin_coord1, origin_coord2, destination_co' at line 1 (1064) (SQLExecDirectW)")  target_table table_staging
2022-05-01 05:27:35,118 - ERROR[etl_loader.SQLServerAccess][sql_connection.merge_execute] - Unable to execute Merge in Staging('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'MERGE table_trips AS target\n            USING table_staging AS source\n          ' at line 1 (1064) (SQLExecDirectW)")
2022-05-01 05:29:47,379 - ERROR[etl_loader.SQLServerAccess][sql_connection.insert_staging_with_df] - Unable to execute INSERT in Staging  ('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'INSERT INTO jobsity.table_staging (origin_coord1, origin_coord2, destination_coo' at line 1 (1064) (SQLExecDirectW)")  target_table table_staging
2022-05-01 05:29:47,409 - ERROR[etl_loader.SQLServerAccess][sql_connection.merge_execute] - Unable to execute Merge in Staging('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'MERGE table_trips AS target\n            USING table_staging AS source\n          ' at line 1 (1064) (SQLExecDirectW)")
2022-05-01 05:33:52,753 - ERROR[etl_loader.SQLServerAccess][sql_connection.insert_staging_with_df] - Unable to execute INSERT in Staging  ('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'INSERT INTO jobsity.table_staging (origin_coord1, origin_coord2, destination_coo' at line 1 (1064) (SQLExecDirectW)")  target_table table_staging
2022-05-01 05:33:52,799 - ERROR[etl_loader.SQLServerAccess][sql_connection.merge_execute] - Unable to execute Merge in Staging('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'MERGE table_trips AS target\n            USING table_staging AS source\n          ' at line 1 (1064) (SQLExecDirectW)")
2022-05-01 05:35:30,801 - ERROR[etl_loader.SQLServerAccess][sql_connection.insert_staging_with_df] - Unable to execute INSERT in Staging  ('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'INTO jobsity.table_staging (origin_coord1, origin_coord2, destination_coord1, de' at line 1 (1064) (SQLExecDirectW)")  target_table table_staging
2022-05-01 05:35:30,829 - ERROR[etl_loader.SQLServerAccess][sql_connection.merge_execute] - Unable to execute Merge in Staging('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'MERGE table_trips AS target\n            USING table_staging AS source\n          ' at line 1 (1064) (SQLExecDirectW)")
2022-05-01 05:36:31,688 - ERROR[etl_loader.SQLServerAccess][sql_connection.insert_staging_with_df] - Unable to execute INSERT in Staging  ('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'INSERT INTO jobsity.table_staging (origin_coord1, origin_coord2, destination_coo' at line 1 (1064) (SQLExecDirectW)")  target_table table_staging
2022-05-01 05:36:31,719 - ERROR[etl_loader.SQLServerAccess][sql_connection.merge_execute] - Unable to execute Merge in Staging('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'MERGE table_trips AS target\n            USING table_staging AS source\n          ' at line 1 (1064) (SQLExecDirectW)")
2022-05-01 05:37:37,694 - ERROR[etl_loader.SQLServerAccess][sql_connection.insert_staging_with_df] - Unable to execute INSERT in Staging  ('HY000', "[HY000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]Incorrect datetime value: '2018-06-31 10:55:12' for column 'datetime' at row 112 (1292) (SQLExecDirectW)")  target_table table_staging
2022-05-01 05:37:37,726 - ERROR[etl_loader.SQLServerAccess][sql_connection.merge_execute] - Unable to execute Merge in Staging('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'MERGE table_trips AS target\n            USING table_staging AS source\n          ' at line 1 (1064) (SQLExecDirectW)")
2022-05-01 05:40:34,521 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error day is out of range for month: 2018-06-31 10:55:12
2022-05-01 05:45:03,333 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error day is out of range for month: 2018-06-31 10:55:12
2022-05-01 05:48:36,285 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error to_datetime() got an unexpected keyword argument 'fomart'
2022-05-01 05:49:00,083 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error time data '2018-05-28 09:03:40' does not match format '%yyyy-%dd-%mm %H:%M:%S' (match)
2022-05-01 05:49:53,003 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error time data '2018-05-28 09:03:40' does not match format '%Y-%d-%m %H:%M:%S' (match)
2022-05-01 05:53:20,026 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error day is out of range for month: 2018-06-31 10:55:12
2022-05-01 05:56:40,516 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error time data 2018-06-31 10:55:12 doesn't match format specified
2022-05-01 05:58:01,124 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error time data 2018-06-31 10:55:12 doesn't match format specified
2022-05-01 05:58:23,218 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error time data 2018-06-31 10:55:12 doesn't match format specified
2022-05-01 05:59:27,220 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error time data 2018-06-31 10:55:12 doesn't match format specified
2022-05-01 06:04:37,814 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error day is out of range for month: 2018-06-31 10:55:12
2022-05-01 06:06:00,566 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error day is out of range for month: 2018-06-31 10:55:12
2022-05-01 06:07:04,011 - ERROR[etl_loader.SQLServerAccess][sql_connection.merge_execute] - Unable to execute Merge in Staging('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'MERGE table_trips AS target\n            USING table_staging AS source\n          ' at line 1 (1064) (SQLExecDirectW)")
2022-05-01 06:09:19,045 - ERROR[etl_loader.SQLServerAccess][sql_connection.merge_execute] - Unable to execute Merge in Staging('42000', "[42000] [MySQL][ODBC 8.0(a) Driver][mysqld-5.7.36-log]You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'MERGE table_trips AS target\n            USING table_staging AS source\n          ' at line 1 (1064) (SQLExecDirectW)")
2022-05-01 16:21:47,261 - ERROR[etl_loader][load.load_data] - name 'MERGE_QUERY_DEL_CONDICION' is not defined
2022-05-01 16:21:47,277 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error name 'MERGE_QUERY_DEL_CONDICION' is not defined
2022-05-01 16:22:03,428 - ERROR[etl_loader][load.load_data] - name 'MERGE_QUERY_DEL_CONDICION' is not defined
2022-05-01 16:22:03,428 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error name 'MERGE_QUERY_DEL_CONDICION' is not defined
2022-05-01 19:26:55,417 - ERROR[etl_loader][elt_loader.get_all] - Export trips with error "['region'] not in index"
